api/services/question/questionService.go

```diff
@@ -13,5 +13,4 @@ type QuestionService interface {
 	Stop(c *gin.Context)
 	Start(c *gin.Context)
 	GetSession(c *gin.Context)
-	GetUserCount(c *gin.Context)
```

Comment:
remove the corresponding docs from the swagger definition to prevent confusion :)